sklearn/neural_network/_base.py

```diff
@@ -175,7 +177,18 @@ def inplace_relu_derivative(Z, delta):
                'relu': inplace_relu_derivative}
 
 
-def squared_loss(y_true, y_pred):
+def _finalize_loss(y_true, y_prob, sample_weight, sample_score):
```

Comment:
I think np.average does the same as this function. What am I missing?

Reply:
I see, you are right, `np.average` also validates the sizes for us.